subsys/logging/CMakeLists.txt

```diff
@@ -9,4 +9,9 @@ if (CONFIG_LOG_FORWARDER_RPC OR CONFIG_LOG_BACKEND_RPC)
     zephyr_library_sources_ifdef(CONFIG_LOG_FORWARDER_RPC log_forwarder_rpc.c)
     zephyr_library_sources_ifdef(CONFIG_LOG_BACKEND_RPC log_backend_rpc.c)
     zephyr_library_sources_ifdef(CONFIG_LOG_BACKEND_RPC_HISTORY_STORAGE_RAM log_backend_rpc_history_ram.c)
+    zephyr_library_sources_ifdef(CONFIG_LOG_BACKEND_RPC_HISTORY_STORAGE_FCB log_backend_rpc_history_fcb.c)
+endif()
+
+if (CONFIG_PARTITION_MANAGER_ENABLED AND CONFIG_LOG_BACKEND_RPC_HISTORY_STORAGE_FCB)
```

Comment:
```suggestion
if(CONFIG_PARTITION_MANAGER_ENABLED AND CONFIG_LOG_BACKEND_RPC_HISTORY_STORAGE_FCB)
```

Reply:
and this should not be here, it should be in the `subsys/partition_manager` folder

Reply:
Done